CHANGELOG.md

```diff
@@ -3,14 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0 - [2024-07-19]
```

Comment:
```suggestion
## v1.0.0 - [2024-07-19]
```
Best to use real semantic versioning from the beginning. 

Any fun format for release names in mind?

Reply:
not yet! Maybe animals-flowers